web/unit_tests/cases/basics.js

```diff
@@ -31,11 +31,11 @@ Modernizr.on('touchevents', function(result) {
     describe('Basic Toggle UI', function() {
 
       beforeEach(function(done) {
-        this.timeout(10000);
+        this.timeout(20000);
```

Comment:
Is this because the tests are taking longer than 10 seconds to run? It feels a little arbitrary. Is it network or environment dependent?

Reply:
I think it's network dependent.  When the tests are run, they're hosted on the local machine, not on BrowserStack servers; a 'tunnel' of sorts is set up that allows the local machine to act as 'localhost' to their remote machines.

Therefore, the quality of net connection affects the needed timeouts... and our connection here locally isn't always the greatest, so periodically the tests fail when the connection acts up.

Some of the recent script changes in test_utils.js do their best to minimize the impact of this, as even initialization tests are far more able to recognize when they've successfully finished all their work, even when it's dependent on script loading, etc.